docs/_config.yml

```
@@ -1,4 +1,4 @@
-exclude: [build, debug, node_modules, spec, src, CNAME, reference-tpl.html, CHANGELOG.md, README.md, LICENSE]
+exclude: [build, debug, node_modules, spec, src, CNAME, reference-tpl.html, CHANGELOG.md, README.md, LICENSE, vendor/bundle, vendor/cache, vendor/gems, vendor/ruby]
```

Do we really need anything from the `vendor` directory? Perhaps we can exclude all of it?
```suggestion
exclude: [build, debug, node_modules, spec, src, CNAME, reference-tpl.html, CHANGELOG.md, README.md, LICENSE, vendor]
```

---

I'm sure on the proper solution but the jekyll website suggested that fix on CI issue https://github.com/Leaflet/Leaflet/pull/8504#issuecomment-1264430063

---

Mentioned in jekyll website:

> The proper solution is to incorporate the default setting for exclude: rather than override it completely:
For versions up to v3.4.3, the exclude: setting must look like following:
```yaml
exclude:
  - Gemfile
  - Gemfile.lock
  - node_modules
  - vendor/bundle/
  - vendor/cache/
  - vendor/gems/
  - vendor/ruby/
  - any_additional_item # any user-specific listing goes at the end
```